app/templates/base.html

```diff
@@ -45,6 +45,10 @@
                     <li class="nav-item">
                         <a class="nav-link" href="/search">Search</a> 
                     </li>
+                    <li class="nav-item">
+                        <button onclick="makejoke()" class="btn btn-link" style='text-decoration:none;'>Make me Laugh</button>
```

Comment:
Move the style to the CSS file.

Reply:
Also the JavaScript, please :) (use `addEventListener`)

Reply:
I  added an event listener and cancel the need in css file